spectrum_io/search_result/msfragger.py

```diff
@@ -16,7 +16,8 @@
 class MSFragger(SearchResults):
     """Handle search results from MSFragger."""
 
-    def read_result(self, tmt_labeled: str) -> pd.DataFrame:
+    def read_result(self, tmt_labeled: str, custom_stat_mods: Dict[str, Tuple[str, float]] = None, 
```

Comment:
please use `stat_mods` and `var_mods` as variables everywhere and fix the type annotations. Likely Optional[Dict[str, str]]